app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -416,6 +416,7 @@ class LuciBuildService {
         'user_agent': const <String>['luci-scheduler'],
       },
       properties: <String, String>{
+        'branch': 'refs/heads/$branch',
```

Comment:
would this be 1.23-xyz or dev|beta|stable?

Reply:
1.23-xyz, this is for Cocoon, not the flutter tool